examples/simple.md

```diff
@@ -0,0 +1,55 @@
+Let's say you have the following Rust binary called `simple`:
+
+```rust,no_run
+use std::env;
+
+fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() == 2 {
+        println!("Hello {}!", args[1]);
+    } else {
+        eprintln!("Must supply exactly one argument.");
+        std::process::exit(1);
+    }
+}
+```
+
+And a trycmd case for your README file:
+
+```rust,no_run
+#[test]
+fn readme() {
+    // Paths are relative to the crate root
+    trycmd::TestCases::new().case("README.md");
+}
+```
+
+You can now write a test for it:
+
+```console
+$ simple World
+Hello World!
+
+$ simple Ferris
+Hello Ferris!
+
+$ simple
+? 1
+Must supply exactly one argument.
+
+```
+
+That's right, the file you're reading right now is a trycmd test!
+
+It uses the trycmd format to run code blocks with the `console` or `trycmd` language:
+
+~~~md
+```console
+$ command ...
+```
+~~~
+
+All other languages are ignored.
```

Comment:
While I'm ok with tying reference-level explanation into the narrative of the example, I feel like this is outside of that narative and should be left to the reference part

Reply:
Fair enough, removed.